specification/cognitiveservices/data-plane/Language/preview/2021-07-15-preview/analyzeconversations.json

```
@@ -249,7 +249,7 @@
         }
       ],
       "properties": {
-        "projectParameters": {
+        "callingOptions": {
           "type": "object",
           "description": "The parameters send to a Question Answering KB."
```

```suggestion
          "description": "The options sent to a Question Answering KB."
```

Or something like that. Using common words will help avoid confusion.

---

Done. Thanks for suggestion.